packages/react/toast/src/Toast.tsx

```diff
@@ -195,34 +204,63 @@ const ToastViewport = React.forwardRef<ToastViewportElement, ToastViewportProps>
       }
     }, [context.isClosePausedRef]);
 
+    const getSortedCandidates = React.useCallback(
+      ({ direction }: { direction: 'forwards' | 'backwards' }) => {
+        const items = getItems();
+        const candidateGroups = items.map((item) => {
+          const itemNode = item.ref.current!;
+          const itemGroup = [itemNode, ...getTabbableCandidates(itemNode)];
+          return direction === 'forwards' ? itemGroup.reverse() : itemGroup;
+        });
+        return (direction === 'forwards' ? candidateGroups : candidateGroups.reverse()).flat();
```

Comment:
it might help here to be more specific:

```suggestion
        const toastItems = getItems();
        const tabbableCandidates = toastItems.map((toastItem) => {
          const toastNode = toastItem.ref.current!;
          const toastTabbableCandidates = [toastNode, ...getTabbableCandidates(toastNode)];
          return direction === 'forwards' ? toastTabbableCandidates.reverse() : toastTabbableCandidates;
        });
        return (direction === 'forwards' ? tabbableCandidates : tabbableCandidates.reverse()).flat();
```